stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -78,17 +92,57 @@ const SecuritiesTable = ({ location }) => {
                 <div className="securities-table-cell">{item.symbol}</div>
                 <div className="securities-table-cell">{item.name}</div>
                 <div className="securities-table-cell">
-                  <Link to={`/inputform/${item.securityId}`}>
-                    <button className="securities-table-button">
-                      <span className="material-icons">edit</span>
+                  <ToolTip message="Edit">
```

Comment:
Again, this might be a thing to add to your big refactoring PR, splitting out this huge return into smaller JSXs.